content/gettingstarted/index.adoc

```diff
@@ -165,6 +165,45 @@ server:
 ----
 {nbsp} +
 
+=== Login Options
+
+Kiali supports three different login types.
+
+*login*: This option allows a user to login into Kiali using a username and password. This is the default option.
+
+*anonymous*: This option removes any login requirement. A user will not be presented the login page and will automatically have access to Kiali without having to present any credentials. 
+
+*openshift*: If you have deployed Kiali on OpenShift you can use this option. With this option users will login into Kiali using the OpenShift OAuth login. What users can access in Kiali will now be based on their user roles in OpenShift using the kubernetes RBAC. See the link:#openshift_user_permissions[section] below for how to grant users access to a specific namespaces.
+
+icon:bullhorn[size=2x]{nbsp} Using the *anonymous* option will leave Kiali unsecured. Anyone who can access the console will have full access to Kiali. If you are using this option you will need to make sure that it is only available on a trusted network and that only trusted users can access it.
+
+The login option can be specified in the `kiali` configmap. For instance, to use the `openshift` login option, the configmap should contain the followin in the `auth` section
+
+[source,yaml]
+----
+auth:
+  strategy: openshift
+----
+
+[#openshift_user_permissions]
+==== OpenShift User Permissions
```

Comment:
It may be useful to add a bit more description about which namespaces will be available to users based on the three login options.  Iiuc,  "login" and "anonymous" get access to the namespaces that Kiali itself has access to.   "openshift" will allow access to the namespaces the user has access to.  As for being able to edit yamls, create route rules, etc (i.e. write/update operations) I'm not exactly sure what is required so it would be nice to see that here.